deployment/customizations.md

```diff
@@ -5,24 +5,117 @@ css_id: customizations
+
+Dynamic delta updates reduce the amount of data sent on each update, while allowing for fine grained control of what version is placed on each ECU.  By using the custom field of the targets metadata, as described in section D.9, the director can be configured to specify a particular version of software for every ECU. Dynamic delta updates allow the director to do  file granular resource tracking, which can save bandwidth by only transmitting the delta of the image.
+
+To send dynamic delta updates, the director would compute the delta as described
+earlier in this section. The computed images would be made available to the primary ECU using the non-standard direction process described in section D.9.
+One drawback of dynamic delta updates is that if many ECUs are updating from the same version, computing the delta of each would result in duplicate computation that could be time consuming or take a lot of memory. A possible solution to this is to use precomputed delta updates.
+
+To send precomputed delta updates the director precomputes various probable diffs and makes these available as images. The director then specifies which precomputed delta image to send to each ECU by using the custom field of targets metadata, as described in section D.9. Precomputing the delta images has the added advantage of allowing these images to be stored on the image repository, which offers additional security against a director compromise.
+
 ## Uptane in conjunction with other protocols
+Implementers MAY use Uptane in conjunction with other protocols already being used
+to send updates to the vehicle.
+
+For example, implementers MAY use SSL / TLS to encrypt the connection between
+primaries and the image and director repositories as well as the time server.
+
+For example, implementers MAY use OMA Device Management (OMA-DM) to send Uptane
+metadata, images, and other messages to primaries.
+
+For example, implementers MAY Unified Diagnostic Services (UDS) to transport Uptane metadata, images, and other messages between primaries and secondaries.
+
+Any system being used to transport images to ECUs needs to modified only to also transport Uptane metadata, and other messages. Note that Uptane does not require network traffic between the director and image repositories and primaries, as well as between primaries and secondaries, to be authenticated.
+
+However, in order for an implementation to be Uptane-compliant, no ECU can cause another ECU to install an image without performing either full or partial verification of metadata. This is done in order to prevent attackers from being able to bypass Uptane, and thus execute arbitrary software attacks that way. Thus, in an Uptane-compliant implementation, an ECU performs either full or partial verification of metadata and images before installing any image (see Section 8 of the Implementation Specification), regardless of how the metadata and images were transmitted to the ECU.
```

Comment:
So, what do I do with this? Should I move it to the standard and if so, where would it go? And, is it just the text highlighted here that needs to move?

Reply:
Still awaiting instructions on this. As I am prepping this copy, it would be very easy to open a pull request and insert it into the standard. But I would need to know exactly where it should go and if its just this text or the whole section.